core/api/service/state/requests/get_keys_paged.hpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_API_SERVICE_STATE_REQUESTS_GET_KEYS_PAGED_HPP
+#define KAGOME_CORE_API_SERVICE_STATE_REQUESTS_GET_KEYS_PAGED_HPP
+
+#include <jsonrpc-lean/request.h>
+
+#include <boost/optional.hpp>
+
+#include "api/service/state/state_api.hpp"
+#include "outcome/outcome.hpp"
+
+namespace kagome::api::state::request {
+
+  class GetKeysPaged final {
```

Comment:
I think a comment explaining or sending to a place explaining what it does would be welcome here.

Reply:
> I think a comment explaining or sending to a place explaining what it does would be welcome here.

https://polkadot.js.org/api/substrate/rpc.html#getkeyspaged-key-storagekey-count-u32-startkey-storagekey-at-blockhash-vec-storagekey

Reply:
> > I think a comment explaining or sending to a place explaining what it does would be welcome here.
> 
> https://polkadot.js.org/api/substrate/rpc.html#getkeyspaged-key-storagekey-count-u32-startkey-storagekey-at-blockhash-vec-storagekey

I would put this one. It has example: https://github.com/w3f/PSPs/blob/psp-rpc-api/psp-002.md#state_getkeyspaged